### ISSUE TITLE
Flipped Half Pie Chart?

### ISSUE THREAD

**Issue by cwestMobile** [danielgindi/Charts#3463]
Is a it possible for the half pie cart to be flipped so the rounded bit is at the bottom and it is flat on the top?

**Comment by petester42** [comment]
Could you provide an image of what you are trying to achieve. I have a hard time picturing it.

**Comment by thierryH91200** [comment]
I think what he wants is that but I think it's not possible

![capture d ecran 2018-05-23 10 12 07](https://user-images.githubusercontent.com/6251463/40411934-28946dca-5e72-11e8-9561-5ea935024f50.jpg)

**Comment by cwestMobile** [comment]
Yeah that is exactly correct, just flip the text and its golden. I haven't figured out a way to do it and was just wondering if it is possible. Cheers.

**Comment by liuxuan30** [comment]
that requires a major change how do we render the text.
